deployment/build.centos.all

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
why file is named `.all`?

Reply:
The .all represents that it's "all architectures", versus ".amd64" or ".arm64" from main builds. Keeps the formatting consistent.